src/Illuminate/Database/Schema/Grammars/ChangeColumn.php

```diff
@@ -133,10 +133,10 @@ protected static function getDoctrineColumnChangeOptions(Fluent $fluent)
         }
 
         if (static::doesntNeedCharacterOptions($fluent['type'])) {
-            $options['customSchemaOptions'] = [
-                'collation' => '',
-                'charset' => '',
-            ];
+            // $options['customSchemaOptions'] = [
+            //     'collation' => '',
+            //     'charset' => '',
+            // ];
         }
```

Comment:
`customSchemaOptions` removed from Doctrine 4: https://github.com/doctrine/dbal/blob/4.0.x/UPGRADE.md#bc-break-removed-support-for-custom-schema-options